tests/Test/Xrefcheck/RedirectChainSpec.hs

```diff
@@ -0,0 +1,113 @@
+{- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
```

Comment:
:warning: The year in this license header is outdated, time to update!





<p align="right" data-meta="generated_by_instant-checks">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>